src/lib/coda_net2/coda_net2.ml

```diff
@@ -61,40 +61,38 @@ module Go_log = struct
   let ours_of_go lvl =
     let open Logger.Level in
     match lvl with
-    | 0 (* Critical gets mapped to error *) | 1 ->
+    | "error" | "panic" | "fatal" ->
         Error
-    | 2 ->
+    | "warn" ->
         Warn
-    | 3 (* Notice gets mapped to info (I can't find any use of notice?) *) | 4
-      ->
+    | "info" ->
         Info
-    | 5 ->
+    | "debug" ->
         Debug
     | _ ->
         Spam
 
   (* there should be no other levels. *)
 
   type record =
-    { id: int64
-    ; time: string
-    ; module_: string [@key "module"]
-    ; level: int
-    ; message: string }
+    { ts: string
+    ; module_: string [@key "logger"]
+    ; level: string
+    ; msg: string
+    ; error: string [@default ""] }
   [@@deriving of_yojson]
 
   let record_to_message r =
     Logger.Message.
-      { timestamp= Time.of_string r.time
+      { timestamp= Time.of_string r.ts
       ; level= ours_of_go r.level
       ; source=
           Some
             (Logger.Source.create
                ~module_:(sprintf "Libp2p_helper.Go.%s" r.module_)
                ~location:"(not tracked)")
-      ; message= r.message
-      ; metadata=
-          String.Map.singleton "go_message_id" (`String (Int64.to_string r.id))
+      ; message= r.msg
+      ; metadata= String.Map.empty
```

Comment:
Should the `error` field be exposed here in metadata if it's not the empty string?

Reply:
ahh good call!